bedrock/exp/templates/exp/firefox/whatsnew/whatsnew-fx71.html

```diff
@@ -0,0 +1,116 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I am not sure why we are including an exp version of the page?

Reply:
We're maintaining a [CRO version](https://www.mozilla.org/en-US/exp/firefox/whatsnew/all/) of the latest /whatsnew page. This PR is updating it from 70 to 71, so that we can run experiments on it once it's out the door.